libs/metrics/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Is it really needed to copy-paste such license files in every submodule?
I hoped to keep a single one in the repo root.

Reply:
All modules that can be distributed separately can have their own licenses, therefore it's a common practice to have a `License` file in all modules that have their own dependencies.

Reply:
For instance, tokio -- https://github.com/tokio-rs/tokio/tree/master/tokio-stream. All crates have their own license files

Reply:
decided to remove these files